nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -98,6 +98,9 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - [Nginx virtual hosts](#opt-services.nginx.virtualHosts) using `forceSSL` or
   `globalRedirect` can now have redirect codes other than 301 through
+
+- `bacula` now allows to configure `TLS` for encrypted communication.
+
```

Comment:
I just noticed this was placed in the middle of another release notes entry.

Reply:
Oh haven't noticed that will make a PR to fix this